src/main/java/fizzbuzz/Rule.java

```diff
@@ -0,0 +1,5 @@
+package fizzbuzz;
+
+public interface Rule {
+    public String check(String number);
```

Comment:
Man braucht in einem Fall String. Würde Integer als normal nehmen. und in dem Fall wo man String braucht einfach Converten. 
Außerdem ist String number als Input Irreführend. 
Außerdem echt schlecht erweiterbar, wenn man für jede neue Rule ne neue Klasse anlegen muss. Gibt irgendwann super viel Clutter im Code. Nen Haufne Klassen die man sich sparen könnte imo